docs/contributing/triaging.md

```diff
@@ -0,0 +1,149 @@
+
+With constant influx of new issues, it's quite essential to prioritize and categorize them efficiently to ensure that the most imporatnt problems are addressed promptly. This document outlines our approach to triaging issues on GitHub, including guidelines for labeling and resolving issues, as well as best practices for maintaining a well-organized and up-to-date issue tracker.
+
+## Labeling Issues
+
+When labeling an issue, choose the label that best describes the issue. Using labels consistently and accurately ensures that issues are easily trackable.
+
+
+### Issue Types
+
+- **bug**: Use for issues that identify bugs causing incorrect or unexpected behavior.
+
+- **dependencies**: Use this label for issues concerning dependencies.
+
+- **documentation**: Use this label for issues related to documentation.
+
+- **library**: Use this label for issues related to core libraries or preparations for future plugin impacts.
+
+
+### Requests and Feedback
+
+- **enhancement**: Use this label for requests for new features or improvements to existing functionalities.
+
+- **feedback**:  Use this label for issues where feedback is requested from the team or our community.
+
+- **help**: Use this label for issues where additional help or contributions are needed.
+
+
+### Special Labels
+
+- **automation**: Use this label for issues related to the CI/CD pipeline or automation
+
+- **duplicate**: Use this label for issues that are duplicates of existing ones.
+
+- **good first issue** : Use this label to highlight issues good for newcomers who want to contribute to Earthaccess.
+
+- **hackathon**: Use this label for issues to be discussed during hackdays.
+
+- **wontfix**: Use this label for issues that won’t be addressed or fixed.
+
+
+## Linking Labels in GitHub Markdown
+
+When referencing a label in a GitHub issue or discussion, it will be  useful to link to the label page to provide additional context and help other members to quickly understand the issue's category.
+
+### Syntax
+
+To link to a label in GitHub Markdown, use the following syntax:
+
+```
+[label name](https://github.com/username/repository/labels/label-name)
+```
+
+### Example
+
+For example, to link to the "hackathon" label in the nsidc/earthaccess repository, you would use the following syntax:
+
+```
+[hackathon](https://github.com/nsidc/earthaccess/labels/hackathon)
+```
+
+!!! tip
+    When linking to labels, it is a good practice to use the label name as the link text, as shown in the example above. This makes it clear to others what label is being referenced, and helps to avoid confusion.
```

Comment:
```suggestion
```

This is a great tip, but if the label syntax is correct, the user shouldn't have to worry about that because GitHub handles the magic :)